spec/workers/podcast_episodes/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe PodcastEpisodes::BustCacheWorker, type: :worker do
+  describe "#perform" do
+    let!(:podcast) { create(:podcast) }
+    let!(:podcast_episode) { FactoryBot.create(:podcast_episode, podcast_id: podcast.id) }
+    let(:worker) { subject }
+
+    before do
+      allow(CacheBuster).to receive(:bust_podcast_episode)
+    end
+
+    describe "when no podcast episode is found" do
+      it "does not call the service" do
+        allow(PodcastEpisode).to receive(:find_by).and_return(nil)
+        worker.perform(789, "/PodCAst/SlUg", "SlUg")
```

Comment:
you don't need to set the expectation on `PodcastEpisode`, you can pass `nil` as the first argument to `perform`